rest-service/manager_rest/test/base_test.py

```diff
@@ -1008,15 +1009,20 @@ def put_deployment_with_labels(self, labels, resource_id=None,
         return deployment
 
     def create_filter(self, filter_name, filter_rules,
-                      visibility=VisibilityState.TENANT, client=None):
-        client = client or self.client
-        return client.filters.create(filter_name, filter_rules, visibility)
+                      visibility=VisibilityState.TENANT, filters_client=None):
+        client = filters_client or self.client
```

Comment:
this won't work because self.client is just the client and filters_client asks to be client.deployments_filters or client.blueprints_filters

Reply:
You're absolutely right. Changed.